ql/src/test/org/apache/hadoop/hive/ql/optimizer/calcite/rules/views/TestHiveAugmentMaterializationRule.java

```diff
@@ -22,6 +22,7 @@
 import org.apache.hadoop.hive.common.ValidReaderWriteIdList;
 import org.apache.hadoop.hive.common.ValidTxnWriteIdList;
 import org.apache.hadoop.hive.common.ValidWriteIdList;
+import org.apache.hadoop.hive.ql.optimizer.calcite.TestRuleBase;
```

Comment:
Unrelated changes; do we need them?

Reply:
These were needed because I moved TestRuleBase to a different package. I will revert it and these changes too.